extensions/Aness6040/Aness_Minecraft_Utils.js

```diff
@@ -0,0 +1,257 @@
+            text: 'Number of Players on [IP] [PORT]',
+            arguments: {
+              IP: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: '127.0.0.1'
+              },
+              PORT: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: 25565
+              }
+            }
+          },
+          {
+            opcode: 'maxPlayers',
+            blockType: Scratch.BlockType.REPORTER,
+            text: 'Max Players on [IP] [PORT]',
+            arguments: {
+              IP: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: '127.0.0.1'
+              },
+              PORT: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: 25565
+              }
+            }
+          },
+          {
+            opcode: 'accountExists',
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: 'Is [PLAYER] a premium account?',
+            arguments: {
+              PLAYER: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'Aness6040'
+              }
+            }
+          },
+          {
+            opcode: 'serverMotd',
+            blockType: Scratch.BlockType.REPORTER,
+            text: '[IP] [PORT] server MOTD',
+            arguments: {
+              IP: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'play.hypixel.net'
+              },
+              PORT: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: 25565
+              }
+            }
+          },
+          {
+            opcode: 'getPlayerTextureUrl',
+            blockType: Scratch.BlockType.REPORTER,
+            text: 'get [PLAYER] [TEXTURELIST] PNG URL',
+            arguments: {
+              PLAYER: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'Aness6040'
+              },
+              TEXTURELIST: {
+                type: Scratch.ArgumentType.STRING,
+                menu: 'textureList',
+                defaultValue: 'face'
+              }
+            }
+          },
+          {
+            opcode: 'getServerIconUri',
+            blockType: Scratch.BlockType.REPORTER,
+            text: '[IP] [PORT] server icon URI',
+            arguments: {
+              IP: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'play.hypixel.net'
+              },
+              PORT: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: 25565
+              }
+            }
+          }
+        ],
+        menus: {
+          textureList: ['face', 'bust', 'cube head','full texture']
+        }
+      };
+    }
+  
+    async isServerOnline(args) {
+      const ip = args.IP;
+      const port = args.PORT;
+  
+      try {
+        const response = await fetch(`https://api.mcsrvstat.us/2/${ip}:${port}`);
```

Comment:
Make sure to check your lint errors-- as they say, please use Scratch.fetch() instead of fetch() for safety reasons.